### PR TITLE
decouple tasks from the release workflow

### PR DESCRIPTION
**Pull request by riprasad**

### CODE REVIEW

File: .github/workflows/update-website.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      - name: Updating Project Website for Release ${{ env.RELEASE_VERSION }}
+        run: |
+          cd website
+          echo "********** Replacing Existing Latest Release JSON File **********"
+          cd _data/registry
+          rm latestRelease.json
+          touch latestRelease.json && curl https://api.github.com/repos/apicurio/apicurio-registry/releases/latest > latestRelease.json
+          
+          echo "********** Saving a copy in the archieve **********"
+          cp latestRelease.json releases/${RELEASE_VERSION}.json
```
Comment (EricWittmann) [inline .github/workflows/update-website.yaml:60]: Should this be `${{ env.RELEASE_VERSION }}` ?

Comment (EricWittmann) [reply .github/workflows/update-website.yaml:60]: Oh nevermind, I see line 49 now.  But still, I'm worried about this line.  If we're using it as a linux ENV var, then maybe it should be this: 
`cp latestRelease.json releases/$RELEASE_VERSION.json`

I can't remember if it's OK to have brackets in there around the variable name.

Comment (EricWittmann) [reply .github/workflows/update-website.yaml:60]: But now I see we use the `${{ env.RELEASE_VERSION }}` syntax later (line 66).  What do you think?  :)

**Review by EricWittmann — APPROVED** [review]

File: .github/workflows/update-website.yaml (hunk 1 above)
Comment (riprasad) [reply .github/workflows/update-website.yaml:60]: > Oh nevermind, I see line 49 now. But still, I'm worried about this line. If we're using it as a linux ENV var, then maybe it should be this:
> `cp latestRelease.json releases/$RELEASE_VERSION.json`
> 
> I can't remember if it's OK to have brackets in there around the variable name.

Both ${RELEASE_VERSION} and $RELEASE_VERSION are acceptable in linux.

Comment (riprasad) [reply .github/workflows/update-website.yaml:60]: > But now I see we use the `${{ env.RELEASE_VERSION }}` syntax later (line 66). What do you think? :)

I am setting RELEASE_VERSION as environment variable at line 49. Every environment variable becomes available in the `env` context.  Now `env` is an object and I am using ${{ env.RELEASE_VERSION }}` to access the field `RELEASE_VERSION` from the `env` context. 

In short, ${RELEASE_VERSION} and $RELEASE_VERSION are Linux syntax. But what if we are using a windows runner, we can't use the Linux syntax. thus we the Jinja2 template i.e. ${{ env.RELEASE_VERSION }}

**Comment by riprasad** [comment]
@EricWittmann Although I have tested the workflows, but I made some last minute changes and did not get a chance to re-test them. Just a heads up

File: .github/workflows/update-website.yaml (hunk 1 above)
Comment (EricWittmann) [reply .github/workflows/update-website.yaml:60]: No problem.  Should we use `${{ env.XXX }}` syntax on line 60 as well then?

**Pull request merged**